### PR TITLE
Add users names to dropdown and display user image

### PR DESCRIPTION
**Pull request by pafff**
Add script to download users.xml file

@bitrut
https://gist.github.com/pafff/35fe216e6bd826537978b09d2da248b8

### CODE REVIEW

File: src/presence_analyzer/script.py [hunk 1]
```diff
@@ -4,6 +4,7 @@
 
 import os
 import sys
+import urllib
```
Comment (bitrut) [inline src/presence_analyzer/script.py]: I definitely recommend using [requests](https://pypi.python.org/pypi/requests/) library next time for things like this. This lib is so good it actually become a standard http client library in Python ecosystem.

**Comment by bitrut** [comment]
Looks good :+1:

**Comment by pafff** [comment]
Updated get_users_file to use requests instead of urllib
@bitrut

File: src/presence_analyzer/script.py [hunk 2]
```diff
@@ -111,3 +112,13 @@ def action_stop(dry_run=False):
         _serve('stop', dry_run=dry_run)
 
     werkzeug.script.run()
+
+
+def get_users_file():
+    r = requests.get('http://sargo.bolt.stxnext.pl/users.xml')
+    if r.status_code == 200:
+        with open(
+                os.path.join('runtime', 'data', 'users.xml'),
```
Comment (bitrut) [inline src/presence_analyzer/script.py]: To big indentation here.

**Comment by bitrut** [comment]
One minor code styling issue, but generally ready to be squashed and merged. :+1:

**Pull request merged**